src/forms/TogglePill.jsx

```diff
@@ -104,15 +33,6 @@ TogglePill.propTypes = {
 
 	/** Whether the pill is toggled on/selected */
 	isActive: PropTypes.bool,
-
-	/** Whether to use input[type="radio"] to limit the selection to one toggle pill in a group */
```

Comment:
Are these props all removed because they are not used by any of the consumers?

Reply:
Yes, we are just dropping support for these and these props and internal state management.